src/solver.cpp

```diff
@@ -231,12 +231,14 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     RegisterWithSolver(registry, ++id, ConvBinWinogradRxS{}, miopenConvolutionAlgoWinograd);
     RegisterWithSolver(registry, ++id, ConvAsmBwdWrW3x3{}, miopenConvolutionAlgoDirect);
     RegisterWithSolver(registry, ++id, ConvAsmBwdWrW1x1{}, miopenConvolutionAlgoDirect);
+#if MIOPEN_BACKEND_OPENCL
```

Comment:
This is wrong. We shall put placeholders (`++id`) if we want remove entries from the list. See comment at the beginning of it.